src/overlays/actors/ovl_Bg_Mori_Idomizu/z_bg_mori_idomizu.c

```diff
@@ -7,8 +13,17 @@
 void BgMoriIdomizu_Init(Actor* thisx, GlobalContext* globalCtx);
 void BgMoriIdomizu_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgMoriIdomizu_Update(Actor* thisx, GlobalContext* globalCtx);
+void BgMoriIdomizu_Draw(Actor* thisx, GlobalContext* globalCtx);
+
+void BgMoriIdomizu_SetupObjectCheck(BgMoriIdomizu* this);
+void BgMoriIdomizu_ObjectCheck(BgMoriIdomizu* this, GlobalContext* globalCtx);
+void BgMoriIdomizu_SetupAction(BgMoriIdomizu* this);
+void BgMoriIdomizu_Action(BgMoriIdomizu* this, GlobalContext* globalCtx);
+
+extern Gfx D_060049D0[];
+
+s16 alreadyLoaded = 0;
```

Comment:
To avoid symbol conflicts you should either give `alreadyLoaded` a name with the `sSystemNameVarName` format or make it `static`. Also this variable seems to be a boolean so you can use true/false instead of 1/0